### ISSUE TITLE
Can we add a new ML project [EXE]

### ISSUE THREAD

**Issue by Abdullah-khan399** [gimseng/99-ML-Learning-Projects#169]
#### Learning Goals
I want to contribute a model that will detect a car in a live stream or video and recognize characters on number plate of the car .

### Exercise Statement
WE used CRNN algorithm for character recognition and EAST( Efficient and accurate scene text detector) algorithm for text detection. It will use the characters and fetch the owners information using RTO API’s . We connected RTO API to database which contains owner details of vehicle.

### Prerequisites 
[Prerequisites, in terms of concepts or other exercises in this repo]
[e.g. random-forest model, stochastic gradient descent, exercise #32]

### Data source/summary:
[Provide a succinct summary of what the data is and where it is from] 
[e.g. This involves covid19 fatality dataset from John Hopkin's website (links..) ]

### (Optional) Suggest/Propose Solutions
[e.g. I have the solution using PyTorch, will be happy to create pull request to include the exercise statement/solution]
[e.g. I think chapter 3 of A. Geron's textbook works out the solution for this exercise]
[e.g. fast.ai's chapter 5 has the perfect solution for this]

### (Optional) Further Links/Credits to Relevant Resources:
[e.g. This exercise and solution's proposal came from a lab session from DL2020]

**Comment by DhruvSanghvi2002** [comment]
I can add new ml project pls assign me the issue
